src/frontend/core/executor/impl/TriangleCountExecutor.cpp

```diff
@@ -424,11 +426,10 @@ static void filter_partitions(std::map<string, std::vector<string>> &partitionMa
 }
 
 void TriangleCountExecutor::execute() {
-    static time_t last_exec = 0;
     schedulerMutex.lock();
     time_t curr_time = time(NULL);
-    if (curr_time < last_exec + 8) {
-        sleep(last_exec + 9 - curr_time);
+    if (curr_time < last_exec_time + 8) {
+        sleep(last_exec_time + 9 - curr_time);
```

Comment:
What does 9 corresponds to here? Can you put a comment in the code explaining the meaning of 9?

Reply:
Ok